### ISSUE TITLE
Option to default to let instead of var

### ISSUE THREAD

**Issue by charlieMonroe** [drkameleon/iswift#74]
I know I'm making a lot of issues here, but I really like the app and am going to use it a lot... :)

You should be using `let` instead of `var` as much as possible. The current code generation will always use vars - this will usually end up with a lot of warnings about not modifying the value.

**Comment by drkameleon** [comment]
Well, this is an interesting one. I know what you mean. However, given Swift's differentiation between `var` ("variable") and `let` ("constant"), I still cannot find a way to know this before hand.

I mean, let's say you declare a varibale in your Objective-C code: `NSString* s;`. So far so good.
If there's no assignment (the variable is not initialized), then it's a `var`. However, what if it's a `NSString* s = @"myString";`? How do we know whether it's going to be altered at some later point in the code? That's tricky.

Your idea of adding an option is not bad. But, wouldn't that create several issues as well (take my example above for instance, where a variable is not initialized. Hence it *should* be a `var`).

I don't know... Food for thought! :)

----

P.S. Please do keep using the app and posting issues! :)

**Comment by charlieMonroe** [comment]
In Swift, you can write code like this:

```
let s: String // No initialization

[ do whatever you want here ]

s = "Hello"
```

You just can't use `s` before it's initialized, but you can pre-declare the variable (but you shouldn't be using an unintialized variable in ObjC either). This is generally useful for if-else statements:

```
let s: String
if self.isDone {
  s = "Done"
}else{
  s = "Not done"
}
```

The difference here is that with `var`, you can re-assign. (if we don't count arrays and dictionaries, where the difference is bigger).

In 90% of cases (IMHO), you are not re-assigning the variables. If you default to `let`, you can get a few errors that you are assigning to a read-only variable, but in my testing, with `var` I'm getting tons of warnings that I have a `var` variable that is never modified...

**Comment by drkameleon** [comment]
Hmm... Good point. I'll see what can be done, or - for now - add an option as you suggested.

**Comment by drkameleon** [comment]
Issue fixed as of the upcoming 1.9.

---

![screen shot 2015-10-03 at 5 39 09 pm](https://cloud.githubusercontent.com/assets/1265028/10263635/bafb9fae-69f5-11e5-88b6-5cc8fae1aa14.png)
